projects/GEDKeeper2/GKUI/Charts/CustomChart.cs

```diff
@@ -214,23 +214,23 @@ public void NavPrev()
                                                  float ang1, float ang2)
         {
             float angval1 = (float)(ang1 * Math.PI / 180.0f);
-            PointF point1 = new PointF(ctX + (float)(inRad * Math.Cos(angval1)),
-                                       ctY + (float)(inRad * Math.Sin(angval1)));
-            PointF point2 = new PointF(ctX + (float)(extRad * Math.Cos(angval1)),
-                                       ctY + (float)(extRad * Math.Sin(angval1)));
+            int px1 = ctX + (int)(inRad * Math.Cos(angval1));

```

Comment:
Reverting back the variables I have reverted **the type** back too! :no_mouth: This is a very bad commit; the fix is around the corner.